Digipost/src/main/java/no/digipost/android/gui/MainContentActivity.java

```diff
@@ -432,8 +432,14 @@ else if (drawerListItems[content].equals(getResources().getString(R.string.drawe
             return true;
 
         }else if (drawerListItems[content].equals(getResources().getString(R.string.drawer_help))) {
-            Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(ApiConstants.URL_HELP));
```

Comment:
Når koden for å åpne ekstern blir såpass stor burde vi også kanskje åpne den i en metode? 
typ: ```java
private void openExternalHelpUrl() {
 //din nye forbedring 
}```